contrib/systemd/system/podman.service

```diff
@@ -8,4 +8,5 @@ StartLimitIntervalSec=0
 [Service]
 Type=notify
 KillMode=process
-ExecStart=/usr/bin/podman system service
+Environment=LOGGING="--log-level=info"
```

Comment:
Why did you do it this way as opposed to just setting it in the ExecStart line?

Reply:
@rhatdan I can see the logging level easily and if desired change to default by commenting out one line.  My user podman.service has a whole list of Environment lines for debugging etc.

Reply:
SGTM